Samples/excel-shared-runtime-scenario/assets/sample.json

```diff
@@ -0,0 +1,61 @@
+[
+    {
+      "name": "office-add-in-ribbon-task-pane-ui",
+      "reponame": "office-add-in-samples",
+      "source": "officedev",
+      "title": "Manage ribbon and task pane UI, and run code on doc open",
+      "shortDescription": "Shows contextual ribbon buttons, hide or show task pane, and how to start add-in on document open.",
+      "url": "https://github.com/OfficeDev/Office-Add-in-samples/tree/main/Samples/excel-shared-runtime-scenario",
```

Comment:
This looks incorrect.

Reply:
The url is correct and goes to Excel shared runtime scenario (unless I'm missing something?)

Reply:
I guess the name confused me, but the URL matches the folder.